games-action/postal/postal-1.0.0.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="An isometric shooter game"
+DESCRIPTION="Postal is a port based on the open sourced original game code
+
+Please note you will need the original game files to play this game.
+
+To play: copy the Linux data folders (res and title) to: 
+/boot/home/config/non-packaged/data/PostalPlus
+"
+HOMEPAGE="https://runningwithscissors.com/games/postal/"
+COPYRIGHT="1997-2003 Running With Scissors"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="1b5d7b9ffe0af4b20ca6825a7471a810ab6a6ef9"
+SOURCE_URI="https://github.com/classiccoding/postal-1-open-source/archive/$srcGitRev.zip"
+CHECKSUM_SHA256="adbfe09eeaf67527703d68c43989ddabecdcfca5bfcdc57857f36b282e90e703"
+SOURCE_DIR="postal-1-open-source-$srcGitRev"
+PATCHES="postal-$portVersion.patchset"
+ADDITIONAL_FILES="postal.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Comment:
Maybe add support for a 32bit build?
```diff
+SECONDARY_ARCHITECTURES="x86"
```

Reply:
It'll probably work but not had much time to test.

Reply:
Build is OK :) 
Is it possible to disable fullscreen startup? (don't have game files to check but it launches fine without errors here on 32bit)

Reply:
I believe you can pass in -windowed on the command line:

postal1 -windowed

Reply:
Works fine in Terminal (not through Deskbar/Tracker (as expected)), now I see the message saying it's missing the game files ;)